### DISCUSSION TITLE
how can i acitivate github student pack?

### DISCUSSION THREAD

**Discussion by ASHWANIashwani** [community/community#110986] [GitHub Education]
i am new to github,i want to acitivate my github student pack

**Comment by SebastienXia** [comment]
Hi, @ASHWANIashwani and welcome to the Github Community !

1) To benefit from the github student pack you need to go to this web site: https://education.github.com/
2) Click on **Join Global Campus**
3) Click on **Yes, I'm Student**
4) You will need to meet all of these requirements before starting your application 
![b](https://github.com/community/community/assets/105004912/7599c586-817d-4fd3-955d-5133b897670b)

5) After all these conditions have been modified or confirmed, all you need to do is apply and follow the process:

Fot more details click [here](https://docs.github.com/en/education)


Sincerely @SebastienXia

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community and to GitHub, @ASHWANIashwani , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
